### PR TITLE
Team Xero and route 1 configurations

### PR DESCRIPTION
**Pull request by Qiangong2**
Added Team Xero
Added Female Trainer
Added Collisions
Added Maps
Started to connect maps
Added Grunt NPCs

### CODE REVIEW

**Comment by Qiangong2** [comment]
This is a huge commit, what do you think?
@bitcraft 
@ShadowApex

**Comment by bitcraft** [comment]
I think we should let the rest of the map making people and writers review it.  I don't see any problems right now, but I haven't given it a good test yet.  It's a lot of content!  Thank you for that.  Let's see if it works with the existing story, then I will review the technical parts.

**Comment by Qiangong2** [comment]
I added the tuxemart shopkeeper from the tuxemon wiki

**Comment by Qiangong2** [comment]
The femaletrainer sprites should not have been changed

File: tuxemon/resources/db/npc/xerogrunt10.json [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
+    "slug": "npc_xerogrunt10",
+    "name_trans": "npc_xerogrunt10"
+}
```
Comment (zapashcanon) [inline tuxemon/resources/db/npc/xerogrunt10.json]: Missing `EOL` here and some others files. You can configure git to prevent this btw.

**Comment by zapashcanon** [comment]
@Qiangong2 I didn't change the size of any image. Looks like the `femaletrainer_*` sprites were already `16*20` before my commit, see [here](https://github.com/Tuxemon/Tuxemon/blob/5c54b5814868f8c1593743f0f7e2ebc268f22e6b/tuxemon/resources/sprites/femaletrainer_back.png).

File: tuxemon/resources/db/locale/en_US.json [hunk 2]
```diff
@@ -205,7 +205,7 @@
     "xero_grunt2": "I'd be willing to let you in if you have a cherry, My tuxemon love them!",
     "xero_grunt3": "Usually they can be found at the bottom of trees",
     "cherry": "You found a cherry",
-    "xero_grunt4": "I see you have a cherry, I'll take that... Now, i'll escort you in."
+    "xero_grunt4": "I see you have a cherry, I'll take that... Now, i'll escort you in.",
```
Comment (zapashcanon) [inline tuxemon/resources/db/locale/en_US.json]: > Now **I**'ll

File: tuxemon/resources/db/locale/en_US.json [hunk 3]
```diff
@@ -202,6 +202,10 @@
     "karrianna_dialog2": "Well... You were the first trainer I've faced...",
     "xero_hideout1": "Xero institute for Tuxemon Research and Advancement ahead.",
     "xero_grunt1": "Hello, welcome to the Xero institute for Tuxemon Research and Advancement, we are currently closed for a remodel.",
+    "xero_grunt2": "I'd be willing to let you in if you have a cherry, My tuxemon love them!",
```
Comment (zapashcanon) [inline tuxemon/resources/db/locale/en_US.json]: > **m**y tuxemon

**Comment by Qiangong2** [comment]
Finally...

**Pull request merged**